CredentialsContainer.js

```diff
@@ -15,7 +15,14 @@ const CREDENTIAL_STORE_TIMEOUT = 0;
 
 export class CredentialsContainer {
   constructor(injector) {
-    this._nativeCredentialsContainer = navigator.credentials;
+    if (typeof navigator.credentials === "object") {
```

Comment:
Not a big deal but could probably use `'credentials' in navigator` vs a `typeof` check?

Reply:
I thought of just checking it as truthy (`if(navigator.credentials)`), but if someone did something stupid like set `navigator.credentials` to a string, it wouldn't fail gracefully. Although, now  that I'm saying that, the code further down would choke on that situation anyway.

I guess it doesn't hurt anything to use the `in` operator.

Reply:
Note on our coding style -- we use single quotes unless there's some very good reason to use double quotes:

`"object"` => `'object'`